packages/block-editor/src/components/list-view/leaf.js

```diff
@@ -32,18 +32,18 @@ const ListViewLeaf = forwardRef(
 		},
 		ref
 	) => {
-		const animationRef = useMovingAnimation( {
-			isSelected,
-			adjustScrolling: false,
-			enableAnimation: true,
-			triggerAnimationOnChange: path,
-		} );
+		// const animationRef = useMovingAnimation( {
+		// 	isSelected,
+		// 	adjustScrolling: false,
+		// 	enableAnimation: true,
+		// 	triggerAnimationOnChange: path,
+		// } );
 
```

Comment:
This is commented out for now while I figure out the right way to handle smoothly moving the dragged item into its final position.